extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -254,6 +291,12 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   onPlpCardClick_(shoppingData) {
+    this.analyticsService_.triggerEvent(
+      StoryAnalyticsEvent.SHOPPING_PLP_CARD_CLICK
+    );
+    this.analyticsService_.triggerEvent(
+      StoryAnalyticsEvent.SHOPPING_PRODUCT_DETAILS_VIEW
+    );
```

Comment:
I am trying to keep track of PDP views.

The PDP is triggered and shown by either

- a shopping tag click
- a product listing page card click

I just want to clarify: the PLP is not the same as the PDP.
The PLP lists only the clickable cards, but we are tracking the page with the actual product description.

This may be redundant:
as PDP views = num shopping tag clicks + num PLP card clicks.

So we can either:

- remove the PDP views event, and just track it by somehow tracking
num shopping tag clicks + num PLP card clicks.

-remove num shopping tag clicks + num PLP card clicks,
and just keep the two PDP views events. 
It sounds like you are leaning towards this approach.

Let me know your thoughts and I can make the change.

Reply:
We had a meeting about this and have decided to go with a different approach:
You can find the details here:
https://docs.google.com/document/d/1YgLIMNDbmUq-rBw4D0FGbpts3xurg0hApKQOC-aVIZo/edit#heading=h.7zq9yzwl3rd